features/admin.console-settings.v1/components/console-roles/console-roles-edit/console-roles-edit.tsx

```diff
@@ -33,6 +36,9 @@ import { useSelector } from "react-redux";
 import ConsoleRolePermissions from "./console-role-permissions";
 import "./console-roles-edit.scss";
 
+
+
+
```

Comment:
```suggestion
```

Can fix in a follow up PR

Reply:
Fixed with [d587fe7](https://github.com/wso2/identity-apps/pull/6938/commits/d587fe7b7259c4a7ddb0ce2652dc1c38c1278380)